ansible-scylla-node/tasks/ssl.yml

```diff
@@ -35,11 +37,11 @@
   block:
     - name: Check if CA crt file exists
       stat:
-        path: "{{ _localhost_cert_path }}/ssl/ca/{{ scylla_cluster_name }}-ca.crt"
+        path: "{{ inventory_dir }}/ssl/ca/{{ scylla_cluster_name }}-ca.crt"
```

Comment:
I guess you wanted a way to provide a custom dir and not to hardcode it ...

Reply:
or wait, just CA has to be in same directory as inventory
and node certs can be elsewhere ?